src/trace_comparison.rs

```diff
@@ -123,6 +123,10 @@ fn compare_json_arrays(arr_1: Vec<Value>, arr_2: Vec<Value>) -> Value {
         return ComparisonResult::new_different(Value::Array(arr_1), Value::Array(arr_2)).into();
     }
 
+    if arr_1.is_empty() && arr_1.is_empty() {
```

Comment:
I think you should check for arr_2 here. Also, not sure if this if is required?

Reply:
Fixed. I also added a test for this check. Without this check, two empty lists will show as [] instead of Same.